crates/mtree2/src/parser.rs

```diff
@@ -53,6 +54,15 @@ impl<'a> MTreeLine<'a> {
                 String::from_utf8_lossy(part)
             );
             if let Ok(keyword) = keyword {
+                if matches!(keyword, Keyword::Wrapped) {
```

Comment:
Keyword implements `PartialEq` and `Eq` so I think this is easier to read:

```suggestion
                if keyword == Keyword::Wrapped {
```